src/components/__tests__/Layout.test.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import Layout from '../Layout';
+
+it('matches output with  snapshot', () => {
+  const layout = shallow(
+    <Layout children={<React.Fragment></React.Fragment>} />
```

Comment:
Close components without children directly.

Also, now you're providing a component that doesn't contain HTML, but you'd like to test whether it appends children properly so make your test reflect that.